cmd/kubeadm/app/constants/constants.go

```diff
@@ -399,15 +402,29 @@ const (
 )
 
 var (
+	// MasterTaint is the taint to apply on the PodSpec for being able to run that Pod on the control-plane
+	// DEPRECATED: https://github.com/kubernetes/kubeadm/issues/2200
+	MasterTaint = v1.Taint{
+		Key:    LabelNodeRoleMaster,
+		Effect: v1.TaintEffectNoSchedule,
+	}
+
+	// MasterToleration is the toleration to apply on the PodSpec for being able to run that Pod on the control-plane
+	// DEPRECATED: https://github.com/kubernetes/kubeadm/issues/2200
+	MasterToleration = v1.Toleration{
```

Comment:
might be we should call this oldControlPlaneToleration (to avoid usage of deprecated worlds in the code, even if temporary)